servicex/expandable_progress.py

```diff
@@ -74,9 +74,10 @@ def __init__(self,
         self.progress_counts = {}
         if display_progress:
             if self.overall_progress or not provided_progress:
-                self.progress = Progress(
+                self.progress = TranformStatusProgress(
                     TextColumn("[progress.description]{task.description}"),
-                    BarColumn(),
+                    BarColumn(complete_style="rgb(114,156,31)",
```

Comment:
This is the muted green color when the bar is progressing.
![image](https://github.com/ssl-hep/ServiceX_frontend/assets/40893179/52c14b0b-5bc9-4f3f-a2e5-6e44b8e9b03b)